### PR TITLE
[W5.11][W12-2]Lee Geng Yu

### PR DESCRIPTION
**Pull request by leegengyu**
An enhancement is made to AddressBook - Level 2 as part of W5.11 LO (Can work with a 2KLoC code base).

### CODE REVIEW

**Comment by auskure** [comment]
This is a good start and helps to give users a sense of time.  Nice work!

**Comment by auskure** [comment]
This would allow users to have a good sense of time when using our program.  It might be even more useful if the time and date are displayed together with the date.  Good work as overall!

**Comment by BearPerson1** [comment]
Maybe a get month method would also help!

<3

**Comment by ABearPerson** [comment]
Maybe a get month method would also help!

as years are too general

**Comment by E0201942** [comment]
It would be cool if the time was noted down when adding contacts so we can have some way to contextualize if we forget how a particular contact is related to us

File: docs/UserGuide.adoc [hunk 1]
```diff
@@ -37,6 +37,7 @@ What's different from AddressBook-Level1:
 * Support for storing address (`a/`) and tags (`t/`)
 * Support for marking a contact detail as 'private' (`pa/`) (`pe/`) (`pp/`)
 * View details of a person (`view` : shows non-private details), (`viewall` : shows all details)
+* View current calendar year every time a command is prompted from the user
```
Comment (okkhoy) [inline docs/UserGuide.adoc:40]: I remember you mentioned your intention in the tutorial for doing this. 
While this enhancement earns you full marks for the LO, I am not sure if you got the full experience of working with an intermediate-size codebase.

**Review by okkhoy — COMMENTED** [review]
At the moment it looks half-baked; since you have explained to me the intention of doing this in the tutorial it makes sense!

**Comment by okkhoy** [comment]
@leegengyu 
Some comments added. Please take note and close the pull request.

**Pull request closed without merging**